sched/group/group_argvstr.c

```diff
@@ -60,16 +61,20 @@
 
 size_t group_argvstr(FAR struct tcb_s *tcb, FAR char *args, size_t size)
 {
+  irqstate_t flags;
   size_t n = 0;
 #ifdef CONFIG_ARCH_ADDRENV
   FAR struct addrenv_s *oldenv;
 #endif
 
+  flags = enter_critical_section();
+
   /* Sanity checks and idle tasks */
 
   if (!tcb || !tcb->group || !tcb->group->tg_info || size < 1 ||
```

Comment:
Which statement in this function depends on group？ could we remove the group check and add critical section to caller？

Reply:
Is it possible that after checking `!tcb->group`, CPU is switched out and `tcb->group->tg_info` triggers the issue?

Currently, only procfs and assert call this function.

Reply:
I see. The direct access to tcb->group has been removed in [this commit](https://github.com/apache/nuttx/pull/12320/files#diff-4c80631b4969dc86c856de89c0a424189374e6284a10a8529662f9e15e3dccd0).

Looks like we do not need `tcb->group` any more. Should I drop the PR directly?

Reply:
pleade remove the group check in this PR，which doesn't a necessary condition for this function